Better README (#559)
bdougie/open-sauced: README.md

<div align="center">
  <br>
  <img alt="Open Sauced" src="https://i.ibb.co/7jPXt0Z/logo1-92f1a87f.png" width="300px">
  <h1>🍕 Open Sauced 🍕</h1>
  <strong>The path to your next Open Source contribution</strong>
</div>
<br>
<p align="center">
  <a href="https://github.com/bdougie/open-sauced/actions?query=workflow%3A%22Node+CI%22">
    <img src="https://github.com/bdougie/open-sauced/workflows/Node%20CI/badge.svg" alt="Node CI">
  </a>
  <a href="https://app.netlify.com/sites/open-sauced/deploys">
    <img src="https://api.netlify.com/api/v1/badges/76a3de8e-270c-4adf-89d5-3a3863da74e6/deploy-status" alt="Netlify Status">
  </a>
  <img src="https://badgen.net/dependabot/bdougie/open-sauced?icon=dependabot" alt="Dependabot Badge">
  <img src="https://img.shields.io/github/languages/code-size/bdougie/open-sauced" alt="GitHub code size in bytes">
  <img src="https://img.shields.io/github/commit-activity/w/bdougie/open-sauced" alt="GitHub commit activity">
  <a href="https://github.com/bdougie/open-sauced/issues">
    <img src="https://img.shields.io/github/issues/bdougie/open-sauced" alt="GitHub issues">
  </a>
</p>

> This is a GraphQL starter project to help people stalk open source repositories and take notes on them. This project uses OneGraph as an interface to the GitHub [GraphQL API](https://developer.github.com/v4/) and Netlify to deploy.

Reviewer: ```suggestion
Open Sauced provides structure onboarding for new contributors to open source. This structure provides a way to track your next contributions by leveraging a unique dashboard built on top of the [GitHub GraphQL API](https://developer.github.com/v4/).
```

[![open-sauced-screencap](https://user-images.githubusercontent.com/5713670/82944481-27abec00-9f50-11ea-85c0-960641717f33.png)
](https://opensauced.pizza)

## 🤝 Contributing

We encourage you to contribute to Open Sauced! Please check out the [Contributing guide](CONTRIBUTING.md) for guidelines about how to proceed.

<img align="right" src="https://i.ibb.co/CJfW18H/ship.gif" width="200"/>

## 📖 Prerequisites

- [Node.js](https://nodejs.org/en/): we recommend using [nvm](https://github.com/nvm-sh/nvm#installing-and-updating) to install the Node version.
- [NPM](https://npmjs.com/): please refer to their [installation guide](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).

## 🖥️ Local development

```
npm install
npm start
```

## 🧪 Test

```
npm test

// to clean snapshots
npm run clean
```

## 📙 Storybook

> Storybook is being leverage to mock out visual React components. A version of it can be found at this [url](https://sauced-components.netlify.com/).

Reviewer: ```suggestion
Storybook is being leveraged to mock out visual React components. The latest version of the design system can be found at this [url](https://sauced-components.netlify.app/).
```
